services/multisig-service/.env

```
@@ -0,0 +1,24 @@
+# Cấu hình Server
+# Cổng này là cổng *bên trong* Docker. Cổng 3001 sẽ được map ở docker-compose
+PORT=${PORT || 3001} 
```

dotenv-linter (4.0.0)

[warning] [SubstitutionKey] The PORT key is not assigned properly

(SubstitutionKey)